client/reader/recent/index.tsx

```diff
@@ -166,21 +169,25 @@ const Recent = () => {
 		} ) );
 	}, [ selectedRecentSidebarFeedId ] );
 
+	useLayoutEffect( () => {
```

Comment:
Using a layoutEffect here prevents the flickering when switching feeds. It forces the `isLoading` state to be updated before the browser paints.